drivers/sensor/bmi160/bmi160.c

```diff
@@ -913,9 +955,24 @@ int bmi160_init(const struct device *dev)
 	/* set default PMU for gyro, accelerometer */
 	data->pmu_sts.gyr = BMI160_DEFAULT_PMU_GYR;
 	data->pmu_sts.acc = BMI160_DEFAULT_PMU_ACC;
+
 	/* compass not supported, yet */
 	data->pmu_sts.mag = BMI160_PMU_SUSPEND;
 
+	/* Start in a suspended state (never turning on the mems sensors) if
+	 * PM_DEVICE_RUNTIME is enabled.
+	 */
+#ifdef CONFIG_PM_DEVICE_RUNTIME
+	pm_device_init_suspended(dev);
+
+	int ret = pm_device_runtime_enable(dev);
+
+	if (ret < 0 && ret != -ENOSYS) {
```

Comment:
just compare with `< 0`. This function no longer return `-ENOSYS`. The documentation is wrong, I'm sending a fix for that.